sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -1954,6 +1954,29 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase with Compilat
         "inputType" -> toSQLType(expression.dataType)))
   }
 
+  def unfoldableInputError(
```

Comment:
just to confirm, there is no ExpressionBuilder throw this error before?

Reply:
I found it. Thank you